pkg/karmadactl/cmdinit/utils/rbac.yaml.go

```diff
@@ -0,0 +1,87 @@
+package utils
```

Comment:
This file's name should have a better consideration.

Reply:
What's your suggestion? @lonelyCZ

Reply:
I think directly `rbac.go` is better.

Reply:
sorry, it's my spelling mistake @lonelyCZ

Reply:
fixed.